src/GovUk.Education.ExploreEducationStatistics.Data.Api.Tests/Services/PermalinkServiceLegacyTests.cs

```diff
@@ -666,129 +668,6 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
         }
     }
 
-    [Fact]
-    public async Task GetLegacy_LegacyLocationsFieldIsTransformed()
```

Comment:
What happened to this test?

Reply:
It's removed because of the internal change made to `PermalinkService.DownloadLegacyPermalink`.

Instead of using `BlobStorageService.SetupDownloadBlobText` to download json and then deserialise it to a LegacyPeramlink, it now uses `BlobStorageService.GetDeserializedJson`  to get a LegacyPermalink, passing the `JsonSerializerSettings` settings.

`BlobStorageService` behaviour is set up to return a `LegacyPermalink` which doesn't require the unit test creating any of the json content which we saw here in the test being removed.

We trust that the settings will do the job of transforming legacy locations which should be ok because `PermalinkResultSubjectMetaJsonConverter` has its own unit test coverage for this.

Reply:
Cool ta